Create Class - Warlock.md

davidggd opened the pull request.

davidggd committed:
Create Class - Rouge.md

davidggd committed:
Update Class - Rouge.md

davidggd committed:
Merge branch 'main' into David's-Changes

davidggd committed:
Corrected Spelling on Rogue

davidggd committed:
Added Warlock

Added the Warlock Class (Rank 1 cards) and changed Rogue's Assasinate to deal +2 damage for each dice rolled.

davidggd committed:
Added Hobgoblin History

ethicalAndroid commented on Class - Warlock.md:
All "Damage" needs to have a specified damage type.
Instead of "+2 extra damage" bonuses to attacks should be "+2 to attack dice rolls"
Damage types should be Capitalized like "Force"

davidggd replied:
Added suggested changes.

ethicalAndroid commented on Advanced History Cards.md:
Specify when in the Hobgoblin Training you add that extra card.
For instance "On pickup, ..." or "At the start of combat, ..."

davidggd committed:
Added Requested Changes to Warlock

Fixed spelling and damage types. Changed Strength for Strength to only deal 4 damage. Clarified Hobgoblin Weapon Training to be on pickup.
